_rules/summary-non-empty-accessible-name-2t702h.md

```diff
@@ -0,0 +1,195 @@
+---
+id: 2t702h
+name: Summary element has non-empty accessible name
+rule_type: atomic
+description: |
+  This rule checks that each `summary` element has a non-empty accessible name.
+accessibility_requirements:
+  wcag20:4.1.2: # Name, Role, Value (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to HTML `summary` elements where all of the following are true:
+
+- the element is [included in the accessibility tree][]; and
+- the element's [DOM tree][] parent is an HTML `details` element; and
+- the element is the first `summary` element of its siblings; and
```

Comment:
Should it instead use "being a [summary for its parent details](https://html.spec.whatwg.org/multipage/interactive-elements.html#summary-for-its-parent-details)"?
Which is exactly the same, but using the HTML definition of it since it exists.